recipes/server.rb

```diff
@@ -27,6 +27,14 @@
     group  'root'
     mode   '0755'
   end
+when 'amazon'
```

Comment:
It's probably a bit easier to add amazon to the existing case statement since it's the same file.

when 'rhel', 'amazon

Reply:
Totally correct, I've made the change as suggested. Thanks.